test/new_relic/http_client_test_cases.rb

```diff
@@ -320,8 +318,7 @@ def test_agent_doesnt_add_a_request_header_to_outgoing_requests_if_xp_disabled
   end
 
   def test_agent_doesnt_add_a_request_header_if_empty_cross_process_id
-    with_config(:'cross_application_tracer.enabled' => true,
-      :'distributed_tracing.enabled' => false,
+    with_config(:'distributed_tracing.enabled' => false,
```

Comment:
Why is DT false here?

Reply:
oh yeah all of these are just like, leftover or whatever and don't matter, I just didn't notice before bc i was focused on the CAT setting, so i didn't really look at other settings. I went through and removed all of these since it doesn't matter if DT is on or not in any of these situations.